main.go

```diff
@@ -124,7 +126,7 @@ func main() {
 	if exportMode {
 		log.Printf("► Exporting organization %s…", cfg.Organization)
 
-		newConfig, err := export.ExportConfiguration(ctx, cfg.Organization, srv, grSrv, licSrv)
+		newConfig, err := export.ExportConfiguration(ctx, cfg.Organization, srv, grSrv, licSrv, throttleRequests)
```

Comment:
Instead of carrying this flag through all the codebase, why not inject it into the service objects? The individual syncing functions should not even know that the service they are using is delaying stuff internally. THis throttling is not an aspect the sync code needs to care about.